{"name": "test002", "status": "failed", "statusDetails": {"message": "AssertionError: assert -1 == 5", "trace": "self = <testDemo.test002.TestClass object at 0x7ff12598e340>\n\n    @allure.feature('allure测试')\n    @allure.story('test002')\n    def test002(self):\n        c = Calc()\n        a = c.jian(2, 3)\n>       assert a == 5\nE       assert -1 == 5\n\ntest002.py:27: AssertionError"}, "attachments": [{"name": "stdout", "source": "fd58b135-4e47-4e13-972a-0d15886636a3-attachment.txt", "type": "text/plain"}], "start": 1639302017491, "stop": 1639302017492, "uuid": "3b30f75d-0ae8-4485-8d8e-2c5ac6bed695", "historyId": "1dbe753821056c3fd3e61cb346c54aad", "testCaseId": "750facfcfedbb613ccc33da634d5afa5", "fullName": "test002.TestClass#test002", "labels": [{"name": "story", "value": "test002"}, {"name": "feature", "value": "allure测试"}, {"name": "suite", "value": "test002"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "MBP.local"}, {"name": "thread", "value": "38604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test002"}]}